modules/end-user-portal/src/appsettings.Development.json

```diff
@@ -6,5 +6,5 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "API_BASEADDRESS": "http://localhost"
+  "API_BASEADDRESS": "http://localhost:5000"
```

Comment:
This can be removed as it doesn't match the config any more

Reply:
Removed.